shared/chat/conversation/messages/dumb.js

```diff
@@ -65,6 +66,160 @@ const textMap: DumbComponentMap<Text> = {
   mocks,
 }
 
+const attachmentBaseMessage = {
+  type: 'Attachment',
+  timestamp: 1479764890000,
+  conversationIDKey: 'cid1',
+  followState: 'You',
+  author: 'marcopolo',
+  deviceName: 'MKB',
+  deviceType: 'desktop',
+  messageID: 0,
+  filename: '/tmp/Yosemite.jpg',
+  title: 'Half Dome, Merced River, Winter',
+  previewType: 'Image',
+  previewPath: null,
+  downloadedPath: null,
+  messageState: 'sent',
+  key: 'foo',
+  senderDeviceRevokedAt: null,
+}
+
+const attachmentMessageWithImg = {
+  type: 'Attachment',
+  timestamp: 1479764890000,
+  conversationIDKey: 'cid1',
+  followState: 'You',
+  author: 'marcopolo',
+  deviceName: 'MKB',
+  deviceType: 'desktop',
+  messageID: 0,
+  filename: '/tmp/Yosemite.jpg',
+  title: 'Half Dome, Merced River, Winter',
+  previewType: 'Image',
+  // $FlowIssue
+  previewPath: require('file-loader!../../../images/yosemite preview.jpg'), // eslint-disable-line
```

Comment:
I added `emit=false` to prevent the file from being emitted, so it still works in hot mode, but not isn't included in prod.

Side effect is that it is also not included in visdiff :(